de-de/python-de.html.markdown

```diff
@@ -119,10 +119,10 @@ False or True #=> True
 "{name} will {food} essen".format(name="Bob", food="Lasagne")
 #=> "Bob will Lasagne kochen"
 
-#Falls dein Python 3 Code auch unter Python 2.5 oder darunter laufen soll, kann das alte Format benutzt werden:
```

Comment:
I think we should delete this - Python3 and python2 are different in ways beyond string interpolation. I think we can delete any statements regarding backwards compatibility

Reply:
I deleted this. But in my opinion it is still important in the first section to write that it exists.

I added the following sentence in the beginning:
_"Beachte hierbei, dass Python 2.7 als deprecated gilt und nicht mehr verwendet werden sollte."_    

What do you think?

Reply:
Yeah okay, that makes sense.

We can replace "deprecated" with veraltet. There are still huge projects running python 2, I think we should say:

"Beachte hierbei, dass Python 2 als veraltet gilt und für neue Projekte nicht mehr verwendet werden sollte."

Reply:
Sounds good, thank you @mxschumacher